sw/device/tests/rv_core_ibex_address_translation_test.S

```diff
@@ -0,0 +1,94 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Converts every ASCII capital letter to a lowercase letter
+ * until a NULL character is encountered.
+ *
+ * @param a0 pointer to the beginning of the null terminated string.
+ */
+  .globl make_lower_case
+  .type repace_foo_with_bar, @function
+make_lower_case:
+  // a0: beginning of string
+  // t0, t1: lower and upper bound of ascii capital letters
+  li t0, 0x41
+  li t1, 0x5A
+
+  addi a0, a0, -1
+make_lower_case_loop:
+  addi a0, a0, 1
+  lb t2, 0(a0)
+
+  // break from loop if the end of the given string is reached
+  beqz t2, make_lower_case_end
+
+  // if not a capital letter skip
+  bltu t2, t0, make_lower_case_loop
+  bgtu t2, t1, make_lower_case_loop
+
+  // makes character lower case
+  xori t2, t2, 0x20
+  sb t2, 0(a0)
+  j make_lower_case_loop
+
+make_lower_case_end:
+  ret
+
+/**
+ * The size of the `make_lower_case` function in bytes.
+ */
+  .global kMakeLowerCaseFnSize
+kMakeLowerCaseFnSize:
+  .4byte .-make_lower_case
+
+
+/**
+ * Replaces the content of the given string with a constant string (`name`)
+ * until a NULL character is encountered in either the given string
+ * or the constant string.
+ *
+ * @param a0 pointer to the beginning of the null terminated string.
+ */
+  .globl get_name
+  .type repace_foo_with_bar, @function
```

Comment:
repace -> replace ?

Reply:
Also, I don't see where this function is actually used.

Reply:
Good spot. I forgot to update the type declaration with the current function names.